lib/FileUploader/FileUploader.js

```diff
@@ -70,7 +73,7 @@ const FileUploader = props => {
             className={`${css.uploadTitle} ${isDropZoneActive ? css.activeUploadTitle : ''}`}
             data-test-title
           >
-            {title}
+            {uploadInProgress ? <Spinner /> : title}
```

Comment:
We should probably also have a little "Uploading" label to drive home the point of what's happening. And remove the `Button` we render below if we're showing the `Spinner`.

May need to convert this component to a class-based one to accomplish that the cleanest way possible, but this render function is getting big enough already so it could be for the best.